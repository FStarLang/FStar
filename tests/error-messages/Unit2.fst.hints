[
  "\u0012U¼úè\u001d\u001eq{c\u000fEy…Èû",
  [
    [
      "Unit2.test3",
      1,
      2,
      1,
      [ "@query", "equation_Prims.nat" ],
      0,
      "97b2ac6b63356cc31fe1f34357af733e"
    ],
    [
      "Unit2.test5",
      1,
      2,
      1,
      [ "@query", "equation_Prims.nat", "equation_Unit2.zat" ],
      0,
      "ff99a34115f80809c0c604ecbd1b29dc"
    ],
    [
      "Unit2.test7",
      1,
      2,
      1,
      [ "@query" ],
      0,
      "9c467a089f9e15785852f3764a9f9a24"
    ],
    [
      "Unit2.__proj__Int__item__i",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query",
        "refinement_interpretation_Tm_refine_a80b1732c37b674c773643729bebf97c"
      ],
      0,
      "a4f5b754ee9c56c6f32ab7faa2d010bb"
    ],
    [
      "Unit2.__proj__Bool__item__b",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query",
        "refinement_interpretation_Tm_refine_51f253259fbb75b1f44c4d01d8397ea7"
      ],
      0,
      "54633d9f8627f87317a836d3133d5bdf"
    ],
    [
      "Unit2.f",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query", "bool_typing",
        "data_elim_Unit2.Bool", "data_elim_Unit2.Int",
        "disc_equation_Unit2.Bool", "disc_equation_Unit2.Int",
        "fuel_guarded_inversion_Unit2.t", "int_typing",
        "primitive_Prims.op_Addition", "primitive_Prims.op_Negation",
        "projection_inverse_BoxBool_proj_0"
      ],
      0,
      "1504b0eda704e0fd1a62be9b416c90f3"
    ]
  ]
]