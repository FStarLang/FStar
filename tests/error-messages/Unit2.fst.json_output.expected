>> Got issues: [
{"msg":["Assertion failed","The SMT solver could not prove the query. Use --query_stats for more details."],"level":"Error","range":{"def":{"file_name":"Unit2.fst","start_pos":{"line":37,"col":21},"end_pos":{"line":38,"col":60}},"use":{"file_name":"Unit2.fst","start_pos":{"line":37,"col":21},"end_pos":{"line":38,"col":60}}},"number":19,"ctx":["While typechecking the top-level declaration `let test6`","While typechecking the top-level declaration `[@@expect_failure] let test6`"]}
>>]
